Makefile

```diff
@@ -44,8 +44,8 @@ add-license-headers: $(GO_ADD_LICENSE)
 	@bash $(GARDENER_HACK_DIR)/add-license-header.sh
 
 .PHONY: format
-format:
-	@go fmt ./cmd/... ./pkg/... ./test/...
+format: $(GOIMPORTS) $(GOIMPORTSREVISER)
+	@bash $(GARDENER_HACK_DIR)/format.sh ./cmd ./pkg ./test
```

Comment:
Analog to: https://github.com/gardener/gardener/blob/8a0ac2fd9684583c65047b5691a73889c9c4fa47/Makefile#L204-L207

Reuses:
https://github.com/gardener/gardener/blob/8a0ac2fd9684583c65047b5691a73889c9c4fa47/hack/format.sh